### PR TITLE
Make propfind work

### PR DESCRIPTION
**Pull request by juliushaertl**

### CODE REVIEW

File: lib/Service/LockService.php [hunk 1]
```diff
@@ -102,15 +103,22 @@ public function __construct(
...
-			$lock = $this->getLockFromCache($node->getId());
+		$propFind->handle(\OCA\FilesLock\AppInfo\Application::DAV_PROPERTY_LOCK, function () use ($node) {
+
+			try {
+				$lock = $this->getLockFromCache($node->getId());
 
-			if ($lock->getUserId() === $this->userId) {
-				return;
+				if ($lock->getUserId() === $this->userId) {
+					return false;
```
Comment (juliushaertl) [inline lib/Service/LockService.php]: @daita Is the lock only identified by the user id or is there a different token as well? 
Maybe we should just return the locking user by default, but if we also want additional details like the time when the lock happened whe need separate properties for that.

Comment (rullzer) [reply lib/Service/LockService.php]: I would say. Userid, displayname and lock time (or remaining) would make sense

Comment (daita) [reply lib/Service/LockService.php]: I will try to improve this, just that it was making the Text App not working as it tries to edit the content when you open the file.

Comment (daita) [reply lib/Service/LockService.php]: fixed!

**Comment by daita** [comment]
added some interaction between front/back end

**Pull request merged**